annotation-processing/src/main/resources/creator.mustache

```diff
@@ -0,0 +1,13 @@
+{{#classDescription.packageName}}
+    package {{{classDescription.packageName}}};
+{{/classDescription.packageName}};
+
+public class {{classDescription.className}} {{#classDescription.interfaceName}} implements {{{classDescription.interfaceName}}} {{/classDescription.interfaceName}}{
```

Comment:
Remove condition about interface. Our class will always implement it. Moreover interface can be hardcoded in this template